src/docs/core/animation.md

```diff
@@ -111,6 +113,14 @@ This can either be a number of milliseconds to delay or triggered by an event. I
 </a-entity>
 ```
 
+Or we could listen for when the element is hovered over:
```

Comment:
`Or we define an animation that listens for when the element is hovered over` to match the previous sentence